docs/troubleshooting.md

```diff
@@ -191,6 +191,13 @@ go_goroutines 20
 ...
 ```
 
+## IMDS
+
+If you're using v1.10, `aws-node` daemonset pod requires IMDSv1 access to obtain Primary IPv4 address assigned to the Node. Please refer to `Block access to IMDSv1 and IMDSv2 for all containers that don't use host networking` section in this [doc](https://docs.aws.amazon.com/eks/latest/userguide/best-practices-security.html) 
```

Comment:
Should this is v1.10.0 ? Since the fix will be made in v1.10.1

Reply:
yeah, makes sense.